### ISSUE TITLE
Bug: Missing request.path parameter in .onRequest

### ISSUE THREAD

**Issue by libersoft-org** [elysiajs/elysia#299]
There is a missing path parameter in ".onRequest":

`app.get('*', (req) => console.log(req.path));`

- returns a correct path

`app.onRequest((req) => console.log(req.path));`

 -> returns: undefined

when listing: console.log(req), there is no parameter for path, only URL.

**Comment by SaltyAom** [comment]
Seems like an invalid type, onRequest is intended to not provide request.path for fast response for example DDOS protection.

`Context.path` is removed with eabe264, available on 0.7.27.

If you want access path, you can use `new URL(request.pathname)`.

Feel free to reopen if the answer is not clear

**Comment by libersoft-org** [comment]
> Seems like an invalid type, onRequest is intended to not provide request.path for fast response for example DDOS protection.
> 
> `Context.path` is removed with [eabe264](https://github.com/elysiajs/elysia/commit/eabe264e7afd76d12ed4a5fd7498d2f89f7afcee), available on 0.7.27.
> 
> If you want access path, you can use `new URL(request.pathname)`.
> 
> Feel free to reopen if the answer is not clear

Unfortunatelly, there is no "request.pathname"... this is how my request object looks like:

```
{
  request: Request (0 KB) {
    method: "GET",
    url: "http://shortener/html/main.html",
    headers: Headers {
      "host": "shortener",
      "connection": "close",
      "user-agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/119.0",
      "accept": "*/*",
      "accept-language": "cs,sk;q=0.8,en-US;q=0.5,en;q=0.3",
      "accept-encoding": "gzip, deflate, br",
      "cache-control": "no-cache",
      "sec-fetch-dest": "empty",
      "sec-fetch-mode": "cors",
      "pragma": "no-cache",
      "priority": "u=4",
      "sec-fetch-site": "same-origin"
    }
  },
  store: {},
  set: {
    cookie: {},
    headers: {},
    status: 200
  }
}
```
